### PR TITLE
Create Hunter page

### PR DESCRIPTION
**Pull request by hunterV1**

### CODE REVIEW

File: _people/Hunter Varghese [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+I am fond of a lot of different types of technoligies, but have no particular
+interest in:
+ - Haskell
+ - Elm
+ - Functional Programming
+ - PostgreSQL
+ - Data Analysis
+ - Build Systems
+
```
Comment (Gayoon1234) [inline _people/Hunter Varghese:22]: this is a comment

**Review by Gayoon1234 — COMMENTED** [review]
I hope that worked

**Comment by Hazelfire** [comment]
Related to #26

**Comment by Hazelfire** [comment]
Closes #26

**Pull request merged**